tests/hugo-site/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "hugo-site",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "serve": "$(npm bin)/hugo server -d out"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "hugo-bin": "^0.70.2"
```

Comment:
This downloads releases directly from https://github.com/gohugoio/hugo/releases so the performance impact should be ok